pkg/vm/builtins/layouts.go

```diff
@@ -0,0 +1,61 @@
+package builtins
+
+import (
+	"fmt"
+
+	"github.com/NethermindEth/cairo-vm-go/pkg/parsers/starknet"
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+)
+
+type LayoutBuiltin struct {
+	Runner  memory.BuiltinRunner
+	Builtin starknet.Builtin
+}
+
+type Layout struct {
+	Name     string
+	RcUnits  uint64
+	Builtins []LayoutBuiltin
+}
+
+func getSmallLayout() Layout {
+	return Layout{Name: "small", RcUnits: 16, Builtins: []LayoutBuiltin{
+		{Runner: &Output{}, Builtin: starknet.Output},
+		{Runner: &Pedersen{ratio: 8}, Builtin: starknet.Pedersen},
+		{Runner: &RangeCheck{ratio: 8, RangeCheckNParts: 8, InnerRCBound: 2 << 16}, Builtin: starknet.RangeCheck},
+		{Runner: &ECDSA{ratio: 512}, Builtin: starknet.ECDSA},
+	}}
+}
+
+func getPlainLayout() Layout {
+	return Layout{Name: "plain", RcUnits: 16, Builtins: []LayoutBuiltin{}}
+}
+
+// TODO: Include Poseidon builtin in this layout
```

Comment:
you can remove the to do you've included it (though its not working yet :p)

Reply:
included and removed 😛